index.html

```diff
@@ -11,6 +11,1053 @@
+                  <img src="https://d.newsweek.com/en/full/1668772/bernie.webp?w=397&h=265&q=90&f=660df23a96579d51aa0d5fe0dfcadda8" alt="">
+                  <p>Politics</p>
+                </div>
+                <div class="card-body">
+                  <h3 class="card-title"><a href="#">Kushner, Ivanka, Don Jr. Resigned to Long Odds; Trump 'Brand' Is, Fight On</a></h3>
+                  <p class="card-text">
+                    "No one should be surprised that Donald Trump is not going to do what the mainstream media wants him to do," says a senior campaign official.
+                  </p>
+                </div>
+              </div>
+
+              <div class="card">
+                <div class="card-img-b-overlay">
+                  <img src="https://d.newsweek.com/en/full/1668772/bernie.webp?w=397&h=265&q=90&f=660df23a96579d51aa0d5fe0dfcadda8" alt="">
+                  <p>Politics</p>
+                </div>
+                <div class="card-body">
+                  <h3 class="card-title"><a href="#">Kushner, Ivanka, Don Jr. Resigned to Long Odds; Trump 'Brand' Is, Fight On</a></h3>
+                  <p class="card-text">
+                    "No one should be surprised that Donald Trump is not going to do what the mainstream media wants him to do," says a senior campaign official.
+                  </p>
+                </div>
+              </div>
+
+              <div class="card">
+                <div class="card-img-b-overlay">
+                  <img src="https://d.newsweek.com/en/full/1668772/bernie.webp?w=397&h=265&q=90&f=660df23a96579d51aa0d5fe0dfcadda8" alt="">
+                  <p>Politics</p>
+                </div>
+                <div class="card-body">
+                  <h3 class="card-title"><a href="#">Kushner, Ivanka, Don Jr. Resigned to Long Odds; Trump 'Brand' Is, Fight On</a></h3>
+                  <p class="card-text">
+                    "No one should be surprised that Donald Trump is not going to do what the mainstream media wants him to do," says a senior campaign official.
+                  </p>
+                </div>
+              </div>
+
+              <div class="card">
+                <div class="card-img-b-overlay">
+                  <img src="https://d.newsweek.com/en/full/1668772/bernie.webp?w=397&h=265&q=90&f=660df23a96579d51aa0d5fe0dfcadda8" alt="">
+                  <p>Politics</p>
+                </div>
+                <div class="card-body">
+                  <h3 class="card-title"><a href="#">Kushner, Ivanka, Don Jr. Resigned to Long Odds; Trump 'Brand' Is, Fight On</a></h3>
+                  <p class="card-text">
+                    "No one should be surprised that Donald Trump is not going to do what the mainstream media wants him to do," says a senior campaign official.
+                  </p>
+                </div>
+              </div>
+              
+              <div class="special-card">
+                <div class="card border my-3" style="width: 18rem;">
```

Comment:
- [ ] Remove this inline styling, this is one of the things affecting the horizontal scroll on your page.